Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -15,6 +15,8 @@ def xorg_recursive_deps
 
   def homebrew_extra_paths
     paths = []
+    # IF ANDROID
+    paths += ["/data/data/com.termux/files/usr/bin", "/data/data/com.termux/files/usr/bin/applets"]
```

Comment:
Add to https://github.com/Linuxbrew/brew/wiki/Android
```sh
ln -s /data/data/com.termux/files/usr/bin/applets/* ~/.linuxbrew/bin/
```

Reply:
Done.